spec/transaction_search_shared_examples.rb

```diff
@@ -11,8 +11,9 @@
       let(:params) { super().merge(account_owners: [3, 4]) }
 
       it "applies the parameter" do
-        expect(assigns(:order_details).where_values)
-          .to be_include("account_users.user_id in ('3','4')")
+        where_values = assigns(:order_details).where_values.reject { |v| v.is_a?(String) }
+        expect(where_values.map(&:to_sql))
+          .to include("`account_users`.`user_id` IN (3, 4)")
       end
```

Comment:
Here's the failure message:

`expected ["\"ORDER_DETAILS\".\"ACCOUNT_ID\" IN (SELECT \"ACCOUNTS\".\"ID\" FROM \"ACCOUNTS\" INNER JOIN \"ACCO...TE\" = :a1", "\"ACCOUNT_USERS\".\"USER_ROLE\" = 'Owner'", "\"ACCOUNT_USERS\".\"USER_ID\" IN (3, 4)"] to include "`account_users`.`user_id` IN (3, 4)"`